google-account-plugin/testSrc/com/google/gct/idea/AccountPluginInitializationComponentTest.java

```diff
@@ -0,0 +1,67 @@
+package com.google.gct.idea;
+
+import org.junit.Test;
+import org.mockito.Mockito;
+
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoMoreInteractions;
+import static org.mockito.Mockito.when;
+
+import com.google.common.collect.Lists;
+import com.google.gct.idea.util.IntelliJPlatform;
+import com.google.gct.idea.util.PlatformInfo;
+
+import com.intellij.mock.MockApplication;
+import com.intellij.openapi.Disposable;
+import com.intellij.openapi.application.Application;
+import com.intellij.openapi.application.ApplicationManager;
+import com.intellij.util.PlatformUtils;
+
+import java.util.List;
+
+public class AccountPluginInitializationComponentTest {
```

Comment:
these seem to duplicate a lot of code across the account and cloud tools init component and tests.